### ISSUE TITLE
Cannot find module 'inherits'

### ISSUE THREAD

**Issue by watson** [AndreasMadsen/dprof#3]
I start the server normally:

```
% cat dprof.json | dprof
server ready on http://localhost:3343
```

But then when I try to open it in a web browser, it complains that it can't find the `inherits` module:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'inherits' from '/Users/watson/.nvm/versions/node/v4.2.1/lib/node_modules/dprof/visualizer'
    at /Users/watson/.nvm/versions/node/v4.2.1/lib/node_modules/dprof/node_modules/browserify/node_modules/resolve/lib/async.js:46:17
    at process (/Users/watson/.nvm/versions/node/v4.2.1/lib/node_modules/dprof/node_modules/browserify/node_modules/resolve/lib/async.js:173:43)
    at ondir (/Users/watson/.nvm/versions/node/v4.2.1/lib/node_modules/dprof/node_modules/browserify/node_modules/resolve/lib/async.js:188:17)
    at load (/Users/watson/.nvm/versions/node/v4.2.1/lib/node_modules/dprof/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/watson/.nvm/versions/node/v4.2.1/lib/node_modules/dprof/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /Users/watson/.nvm/versions/node/v4.2.1/lib/node_modules/dprof/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:82:15)
```

**Comment by watson** [comment]
Ah, it seems this have been fixed in 6465fc0. So if I install using `npm i AndreasMadsen/dprof && npm i -g AndreasMadsen/dprof` it works.

**Comment by AndreasMadsen** [comment]
Published a new version.
